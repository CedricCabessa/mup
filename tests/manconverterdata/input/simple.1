.TH EXAMPLE 1
.SH NAME
example \- an example
.SH SYNOPSIS
example
.RB [ -f ]
.RB [ -l ]
