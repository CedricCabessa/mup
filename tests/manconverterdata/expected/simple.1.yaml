patterns:
    - NAME
    - example\s+-\s+an\s+example
    - SYNOPSIS
    - -f
    - -l
